api/src/main/java/org/openmrs/module/fhir2/providers/r4/MedicationRequestFhirResourceProvider.java

```diff
@@ -48,4 +57,29 @@ public MedicationRequest getMedicationRequestByUuid(@IdParam @NotNull IdType id)
 		return medicationRequest;
 	}
 	
+	
+	@Search
+	@SuppressWarnings("unused")
+	public IBundleProvider searchForMedicationRequests(
```

Comment:
@ibacher do you mean this for r4 ?

Reply:
Yep!

Reply:
@ibacher does it satisfy the requirements?